single-line-diagram/single-line-diagram-core/src/main/java/com/powsybl/sld/svg/DefaultSVGWriter.java

```diff
@@ -1226,17 +1216,22 @@ private void drawNodesInfos(Element root, VoltageLevelGraph graph,
         double yPos = graph.getY() - layoutParameters.getVoltageLevelPadding().getTop() + graph.getHeight() + CIRCLE_RADIUS_NODE_INFOS_SIZE;
 
         double xShift = graph.getX() + xInitPos;
-        for (ElectricalNodeInfo node : initProvider.getElectricalNodesInfos(graph)) {
-            String idNode = metadata.getSvgParameters().getPrefixId() + "NODE_" + node.getBusId();
+        for (NodeInfo nodeInfo : initProvider.getNodesInfos(graph)) {
+            String userDefinedId = nodeInfo.captions().stream()
+                .filter(c -> c.type().equals("userDefinedId"))
```

Comment:
I realized that in the previous ElectricalNodeInfo class, we had a constructor with a `userDefinedId` String parameter. Where is the equivalent in the new version of the code?

Another minor remark: this `userDefinedId` variable is only used for `metadata.addElectricalNodeInfoMetadata(...(idNode, userDefinedId))`. I think it would be clearer to declare and assess `userDefinedId` just before that line.

Reply:
This userDefinedId was asked by a specific user, who didn't use it at the end. I'll check it again, but to my knowledge this is not used and could be deprecated/broken.

Reply:
This id was added to address this issue https://github.com/powsybl/powsybl-diagram/issues/287.
If it is not used, it may be removed.

Reply:
The equivalent would be to create a NodeInfo.Caption with type "userDefinedId" in "getNodesInfos" in LabelProvider
Should I keep this behaviour ?

Reply:
No, you can remove this behaviour we don't want userDefinedId anymore (and in fact the equivalent would be to add an optional userDefinedId for each caption; it was used to link each value with a measurement id)

Reply:
fixed